modular_skyrat/modules/mapping/blueshift/code/areas.dm

```diff
@@ -0,0 +1,23 @@
+/area/station/common/gaskiosk
+	name = "\improper Internals Kiosk"
+	icon_state = "vacant_commissary"
+
+/area/station/commons/dorms/room5
+	name = "\improper Dorms Room 5"
+	icon_state = "room1"
+
+/area/station/commons/dorms/room6
+	name = "\improper Dorms Room 6"
+	icon_state = "room2"
+
+/area/station/commons/dorms/room7
+	name = "\improper Dorms Room 7"
+	icon_state = "room3"
+
+/area/station/commons/dorms/room8
+	name = "\improper Dorms Room 8"
+	icon_state = "room4"
+
+/area/station/commons/dorms/vacantroom
+	name = "\improper Vacant Dorms Room"
+	icon_state = "vacant_room"
```

Comment:
Please put this in `**/modules/mapping/area/**`.
I want to get away from maps having their own modules because its needlessly messy and stupid.

Reply:
Moved